<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskInfoWindow</class>
 <widget class="QWidget" name="TaskInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>575</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(194, 200, 210);
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>45</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>600</width>
              <height>40</height>
             </size>
            </property>
            <widget class="QPushButton" name="btnBack">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>72</width>
               <height>42</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">
QPushButton
{
 width:70px;
 height:40px;
  color: black;
  background-color: #cecece;
  border: 1px solid #888888;
  border-radius: 5px;
}

QPushButton:hover
{
  color: black;
  background-color: #eeeeee;
}

QPushButton:pressed
{
  color: white;
  background-color: #bbbbbb;
}</string>
             </property>
             <property name="text">
              <string>BACK</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>50</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>60</y>
              <width>41</width>
              <height>23</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  color: white;
  background-color: #1362AB;
  border-width: 0px;
  border-radius: 5px;
}

QPushButton:hover
{
  color: white;
  background-color: #1B91FF;
  border-width: 0px;
  border-radius: 5px;
}

QPushButton:pressed
{
  color: white;
  background-color: #074680;
  border-width: 0px;
  border-radius: 5px;
}</string>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton_4">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>240</y>
              <width>51</width>
              <height>23</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
  background-color:transparent;
  border-radius: 10px;
   image: url(:/Images/images/btn_switch_on.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>491</width>
              <height>271</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
background-color:white;
border:: 1px solid black;
border-radius:20px;
}</string>
            </property>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>230</y>
               <width>301</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Add me as watcher to tickets created by others</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="chkWatcher">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>220</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox {
    spacing: 0px;
}

QCheckBox::indicator {
    width: 50px;
    height: 30px;
}

QCheckBox::indicator:unchecked {
    image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:unchecked:pressed {
        image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:checked {
        image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/Images/images/btn_switch_on.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QCheckBox" name="chkActive">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>10</y>
               <width>51</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox {
    spacing: 0px;
}

QCheckBox::indicator {
    width: 50px;
    height: 30px;
}

QCheckBox::indicator:unchecked {
    image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:unchecked:pressed {
        image: url(:/Images/images/btn_switch_off.png);
}

QCheckBox::indicator:checked {
        image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/Images/images/btn_switch_on.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/Images/images/btn_switch_on.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>110</y>
              <width>111</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Tickets visibility</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="edName">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>60</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Active</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>300</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 85, 0)</string>
            </property>
            <property name="text">
             <string>Statistics</string>
            </property>
           </widget>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>330</y>
              <width>491</width>
              <height>111</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:white;
border-radius:20px
</string>
            </property>
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>10</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>Last 30 Days</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>This Month</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>70</y>
               <width>91</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>All Time</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_13">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>10</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>00:00:00 /</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_14">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>10</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 85, 0)</string>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_15">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>40</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 85, 0)</string>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_16">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>70</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 85, 0)</string>
             </property>
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_17">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>40</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>00:00:00 /</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_18">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>70</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>00:00:00 /</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>70</y>
              <width>47</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>200</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 85, 0)</string>
            </property>
            <property name="text">
             <string>Delete project</string>
            </property>
           </widget>
           <widget class="QWidget" name="horizontalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>100</y>
              <width>160</width>
              <height>32</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
             <item>
              <widget class="QLabel" name="lblTime">
               <property name="styleSheet">
                <string notr="true">color:rgb(255, 85, 0)</string>
               </property>
               <property name="text">
                <string>All Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_6">
               <property name="styleSheet">
                <string notr="true">QPushButton
{
  color: black;
  width:20px;
  height:20px;
  background-color:transparent;
  border-width: 0px;
  border-radius: 3px;
}
QPushButton:hover
{
  background-color: #A9A9A9;
  
}</string>
               </property>
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>150</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Workers</string>
            </property>
           </widget>
           <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>140</y>
              <width>301</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout"/>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
